vscode/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "qsharp-lang-vscode-dev",
   "displayName": "[DEV BUILD] Azure Quantum Development Kit (QDK)",
-  "description": "Q# Language Support",
+  "description": "Quantum Language Support",
```

Comment:
I'd lean towards _adding_ OpenQASM, not removing Q#. When folks go to the marketplace and search on either, we want our extension to show up.